app/services/resource_service.rb

```diff
@@ -14,16 +14,32 @@ def create(resource_class, params)
     @resource_class = resource_class
     @resource = resource_class.new(params)
     @resource.data_provider = data_provider
-
-    # skip create if already exists
+    # skip create if record already exists and new record has the same attribute values as the new record and the record was not provided by maz
```

Comment:
Zeile ist etwas zu lang

Reply:
"skip create if record already exists and new record has the same attribute values as the **old record** and the record was not provided by maz"